packages/flutter_test/lib/src/event_simulation.dart

```diff
@@ -66,22 +65,23 @@ class KeyEventSimulator {
         map = kWindowsToPhysicalKey;
         break;
       case 'web':
-      // web doesn't have int type code
-        return null;
+        // web doesn't have int type code
+        return -1;
```

Comment:
fyi @gspencergoog

Reply:
Yes, I saw this when I did my review, and it simplifies other code here, but doesn't change behavior, so a net win.